docs/reference/modules/attachment.md

```diff
@@ -238,6 +238,12 @@ The `options` parameter is optional. If `options.permissions` is explicitly set
 ### `crop(req, _id, crop)`
 This method takes the image specified by the `_id` of an existing attachment, copies it to the uploadfs specified temporary location, applies the crop, and then saves it back into uploadfs storage. The `crop` parameter takes an object with `top`, `left`, `width`, and `height` properties. The passed values should be unitless, but must be JavaScript numbers, not strings.
 
+### `addFieldGroup(group)`
```

Comment:
`addFileGroup` right?

Reply:
Dammit! My "muscle memory" keeps defaulting to that.